setup.py

```diff
@@ -102,39 +102,39 @@
         "setuptools_scm[toml]>=6.2",
     ],
     install_requires=[
-        # --- core dependencies
-        f"fontTools{FONTTOOLS_VERSION}",
-        "freetype-py!=2.4.0",  # Avoiding 2.4.0 due to seg-fault described at
-        # https://github.com/googlefonts/fontbakery/issues/4143
-        "munkres",  # For interpolation compatibility checking (fontTools dependency)
-        "opentypespec",
+        "axisregistry>=0.3.0",
+        "beautifulsoup4",
+        "beziers>=0.5.0",  # Uses new fontTools glyph outline access
+        "cmarkgfm",
+        "collidoscope>=0.5.2",  # 0.5.1 did not yet support python 3.11
+        # (see https://github.com/googlefonts/fontbakery/issues/3970)
+        "defcon",
+        "dehinter>=3.1.0",  # 3.1.0 added dehinter.font.hint function
+        "fontTools[ufo,lxml,unicode]>=4.36.0",  # allows for passing location to glyphsets
+        "font-v",
+        "gflanguages>=0.3.0",  # there was an api simplification/update on v0.3.0 (see https://github.com/googlefonts/gflanguages/pull/7)
+        "glyphsets>=0.5.0",
+        "lxml",
+        "munkres",  # for interpolation compatibility checking
```

Comment:
This PR is bringing back all the contents of the **setup.py** from before a recent commit this week that implemented profile-specific **extras**. I can fix this one for you in a followup PR, if you are OK with this.

Reply:
well... actually not a followup PR, but a new PR to replace this one.

Reply:
OK. I will deal with this one today.